composer.json

```diff
@@ -22,17 +22,23 @@
         "issues": "http://github.com/liip/RMT/issues"
     },
     "require": {
-        "php": ">=5.3.9",
+        "php": "^5.6|^7.0",
         "symfony/console": "~2.3|^3.0",
         "symfony/yaml": "~2.3|^3.0",
         "symfony/process": "~2.3|^3.0",
         "vierbergenlars/php-semver": "~3.0",
         "sensiolabs/security-checker": "~2.0|^3.0|^4.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^5.6.8|^6.0"
```

Comment:
Ah yes, but this is where we use the power of composer. This version constraint will allow composer to determine which version to use. As you can see on the travis build php 5.6 will use phpunit 5.7.21 and php 7.x uses phpunit 6.2.3.

This way we can use the forward compatibility layer of phpunit and support phpunit 5.x and 6.x (and thus also php 5.6 and 7.x) at the same time